packages/zwave-js/src/lib/controller/Controller.ts

```diff
@@ -897,7 +905,7 @@ export class ZWaveController
 		const apiCaps = await this.driver.sendMessage<
 			GetSerialApiCapabilitiesResponse
 		>(
-			new GetSerialApiCapabilitiesRequest(this.driver),
+			new GetSerialApiCapabilitiesRequest(this.driver, {mysteryValue: 3}),
```

Comment:
This is a holdover from when I couldn't get inclusion working. SimplicityStudio adds a mystery single byte field to this command with value 3. it's not documented, and I think the inclusion problems were security related. Probably should back this out.